### PR TITLE
Add translation for working guideline

### PR DESCRIPTION
**Pull request by frasun**
#### Changelog:
- translate existing guides to English

### CODE REVIEW

File: workflow-guideline.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## Working with Git
+
+By definition commits should address small, meaningful and complete pieces of the feature that is currently under development. The list of commits attached to the Pull Request should be a step-by-step guide on the approach that was taken to solve a problem.
+
+Remember to include the general change log in the Pull Request description. It will serve as an explanation to the reviewers why those changes were needed.
+
+### Things considered a good practice when working with Pull Requests:
+- Stick to the naming convention that the team agreed on e.g. `feature/nut-123-style-appointment-screen/`
+- Respect the template for opening new Pull Requests if one is provided. Otherwise try to stay consistent with rest of the team
```
Comment (pokorson) [inline workflow-guideline.md]: Maybe we should emphasize that we prefer to always have PR template if possible?

File: workflow-guideline.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Make sure that your Pull Request is in fact ready and could be merged to master branch as is. Don’t force others to waste their time on finding typos or silly mistakes
+- It is acceptable to use ‚work in progress’ Pull Requests to satisfy different needs during development process. In that case attach the `WIP` signature to the title and do not ask for review until the Pull Request is good to go
```
Comment (pokorson) [inline workflow-guideline.md]: Github has this new `Draft` feature, maybe it's worth to mention it here?

**Comment by pokorson** [comment]
@frasun now I started to wonder is it worth to maintain this in 2 languages? Initially I've written it in polish but I guess we can drop that one and use this one instead.

File: workflow-guideline.md (hunk 1 above)
Comment (frasun) [reply workflow-guideline.md]: Good idea, added a note to create one if not used

File: workflow-guideline.md (hunk 2 above)
Comment (frasun) [reply workflow-guideline.md]: Right, have not used it but included as a second way next to WIP prefix

**Comment by frasun** [comment]
@pokorson right, I guess it was only waiting for the translation to come since all of our docs are in English 🙂

**Pull request merged**